### DISCUSSION TITLE
daily maps

### DISCUSSION THREAD

**Discussion by maitra** [matteason/daily-cloud-maps#4] [General]
Just came across this project and it is great, thank you!! Just wondering: why have only daily updates, perhaps 4 or 6/day might be more helpful, to see storms, etc that develop, or is there a technological limitation in the download? Thank you again for this map!

**Comment by matteason** [comment]
Hi @maitra - so sorry I missed this before. 

I would love for the updates to be more frequent. Unfortunately the only free data source I could find for whole-Earth images was NOAA-20 VIIRS, which only provides data via an API once per day for any particular region, so I have to wait for the whole Earth to be covered before I can generate a new image. 

There are other sources which provide more frequent updates but unfortunately they don't cover the whole Earth and they have other features like grid lines and country borders which would need a lot more post-processing to filter out. I'm always keeping an eye out for new data sources though, so I will update this project if something comes along that makes more frequent updates possible

**Reply by maitra** [reply]
Thanks very much for this detailed explanation, and thanks again for this very helpful resource!

**Reply by matteason** [reply]
Ok, so promped by this I've had another hunt for data sources and I _think_ I've found one that will provide images every three hours (so 8/day), which would be a huge improvement. As an added bonus, the images are captured simultaneously from geostationary satellites, which means the banding present at the moment (because the NOAA satellite is polar orbiting, so captures the Earth in strips) won't be there any more. 

The source images will need to be processed differently to how they are at the moment so it won't be a drop-in replacement, but I'll let you know how I get on!

**Reply by maitra** [reply]
Excellent, thank you very much for all the hard work!!

**Comment by matteason** [comment]
Update on this: https://github.com/matteason/daily-cloud-maps/discussions/9
